consensus/core/src/authority_node.rs

```diff
@@ -201,18 +217,19 @@ where
         let block_manager =
             BlockManager::new(context.clone(), dag_state.clone(), block_verifier.clone());
 
+        let leader_schedule = Arc::new(LeaderSchedule::from_store(
+            context.clone(),
+            dag_state.clone(),
+        ));
```

Comment:
For additional peace of mind I would also feature flag guard the inner workings of the `LeaderSchedule::from_store` . I understand that we write the default scores in store and in the end things will be the same until we actually start populating any scores, but I would feel better if we could eliminate any risk here before the flag is flipped